webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -490,6 +490,31 @@ def update_corr_param_resp(
         parameters: List[str],
         corr_param_resp: Dict,
     ) -> Optional[Dict]:
+        """
+        update_corr_param_resp is a callback function responsible for handling
+        which response and parameter are active.
+
+        The correlation plots display correlation between a given response and
+        paramater (at an index, see `corr_xindex`) for a chosen ensemble. Hence
+        the need to keep track of which of potentially many selected responses
+        and parameters, respectively, is currently active.
+
+        It should be triggered upon
+        - modifying the selection of parameters or responses
+        - clicking on the heatmap, effectively choosing an active response and
+          parameter
+
+        To avoid saying "response or parameter", we refer to either
+        collectively by "variable".
+
+        When the first variable is selected, they become active.
+        Further selection of variables does not modify the active variable.
+        Deselection of the active variable makes the first selected variable
+        active, otherwise deselection does not affect the active state.
+
+        Clicking on the heatmap makes the resoponse and parameter active that
```

Comment:
makes the "combinations of " ... if you agreed with the previous suggestion.

Reply:
sure ^^ and done.